packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -72,6 +75,71 @@ export function SelectWalletModal({
     [disconnectWallet],
   );
 
+  const onConnectMagicEdenWallet = useCallback(async () => {
+    if (network === "testnet") {
+      const unsupportedNetworkError = new Error(
+        "Magic Eden wallet is not supported on testnet",
+      );
+      onError(Wallet.MAGICEDEN, unsupportedNetworkError);
+      return false;
+    }
+
+    try {
+      setErrorMessage("");
+      const magicEden = await getMagicEdenAddress(network);
+      if (!magicEden || magicEden.length < 1) {
+        disconnectWallet();
+        throw new Error("Magic Eden via Ordit returned no addresses.");
+      }
+
+      // Magic Eden provides a segwit address by default for sending and receiving payments
+      // Imported xverse wallets will return a p2sh address for payments by default instead
+      const paymentAddress = magicEden.find(
+        (walletAddress) =>
+          walletAddress.format === "segwit" ||
+          walletAddress.format === "p2sh-p2wpkh",
+      );
+
+      const taproot = magicEden.find(
```

Comment:
same thing, lets make it consistent. 
```suggestion
      const ordinalAddress = magicEden.find(
```